index.html

```diff
@@ -16,7 +16,7 @@
 					<a href="#" class="hover_effect">George M'sapenda</a>
 				</div>
 				<button class="btn">
-					<img src="images/Menu_icon.png" alt="menu bar icon">
+					<img src="/images/Icon - Menu.png" alt="menu">
```

Comment:
- [ ] Kindly make sure you use the correct path for your hamburger menu image.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200908720-fc93d74a-9cea-447a-a42d-2509f97caa37.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200908952-81511921-cb95-49f3-b76d-035fb3aa7d6d.png)